docs/Wdio_Utility.md

```diff
@@ -85,12 +85,24 @@ Then, to assist with testing, the TerraService provides the custom assertions `a
 - `Terra.viewports(name)` takes the viewport key name(s) and returns an array of { height, width } objects representing the respective terra viewport size(s).
     - Use this function to resize the browser or pass the viewport sizes to the accessibility and visual regression commands.
     - By default returns all viewports if not name key are provided.
-
+- `Terra.should.beAccessible()` convenience method that injects an axe test. Takes the same arguments as the `axe()` utility. See [beAccessible-spec.js](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/beAccessible-spec.md) for examples.
+- `Terra.should.matchScreenshot(name, options)` convenience method that injects a screenshot test. See [matchScreenshot-spec.js](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/matchScreenshot-spec.md) for example usage..
 ```js
 // These globals are provide via the Terra Service
 /* global browser, describe, it, expect, viewport */
 
 describe('Basic Test', () => {
+  before(() => browser.url('/test.html'));
+
+  Terra.should.beAccessible();
+  Terra.should.matchScreenshot();
+
+  it('custom test', () => {
+    expect('something').to.equal('something');
+  });
+});
+
+describe('Advanced Test', () => {
```

Comment:
Wondering if we give them the information on the Terra object then give the basic example, then explain the accessible() and matchReference() assertions & give the advance to break it up and emphasis the Terra global?

Otherwise looks good.

Reply:
👍  updated